### PR TITLE
WIP: Add missing fragmentDensityMap validations

### PR DESCRIPTION
**Pull request by byumjin**
#1097

### CODE REVIEW

File: layers/buffer_validation.cpp [hunk 1]
```diff
@@ -1370,6 +1370,31 @@ bool CoreChecks::PreCallValidateCreateImage(VkDevice device, const VkImageCreate
                         "maxFramebufferHeight");
     }
 
+    uint32_t ceilingWidth = (uint32_t)ceil((float)device_limits->maxFramebufferWidth /
```
Comment (mark-lunarg) [inline layers/buffer_validation.cpp]: @byumjin, can you please change the local variable names to `lower_snake_case`?  Function names should be `UpperCamelCase`.

Comment (byumjin) [reply layers/buffer_validation.cpp]: Yes, I will change them right away.

**Comment by mark-lunarg** [comment]
@byumjin -- Please rebase your changes on Master and squash your commits.  Also, each of the validation checks will require a test.  These *should* be in a separate commit.  Sorry I didn't notice this sooner!

**Comment by byumjin** [comment]
I re-based my repo, and I'm adding the codes for testing.
But, the logs are quite messed up.
I think it would be better to start with new repo after I finished to make testing codes.

By the way, is there way to test codes what I added only?
Because I need to wait for my tests until many tests are passed

**Comment by mark-lunarg** [comment]
@byumjin, you can redo your local branch and then force-push it to github, and it will do the right thing, or you can close and create a new PR, either one is fine!
And yes, you can run individual tests by adding the following argument when invoking the executable:

`> vk_layer_validation_tests --gtest_filter=VkLayerTest.YourTestNameHere`

**Comment by byumjin** [comment]
@mark-lunarg, thank you! the gtest filter works well.  By the way, I also would like to test my build on android. And I found this out to run the test app
>adb shell am start com.example.VulkanLayerValidationTests/android.app.NativeActivity
>
Is there a similar way to filter the tests for android?

And, it seems like every building request in pull request is stuck(pending forever), now.

**Pull request closed without merging**